packages-content-model/roosterjs-content-model-dom/lib/modelToDom/contentModelToDom.ts

```diff
@@ -27,6 +27,7 @@ export function contentModelToDom(
     onNodeCreated?: OnNodeCreated
 ): DOMSelection | null {
     context.onNodeCreated = onNodeCreated;
+    context.regularSelection.isReverted = isSelectionReverted(doc.getSelection());
```

Comment:
selection in content model may not be the same with current selection, so here we can't use doc.getSelection().

Actually when create Content Model, I think the isReverted info is lost, because we only have "isSelected" in Content Model but we don't store if it is reverted.

Do we need this info when write back from Content Model? If not, we can always set to false and leave a TODO here.

Reply:
Ok, lets remove it. 
The only problem here is that if the selection was reverse of the document, after model to dom, the reverse order could be lost. But it is probably a small issue.